src/components/autocomplete-select/autocomplete-select.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useMemo, useRef, useState } from 'react';
+
+import {
+  FlatList,
+  GestureResponderEvent,
+  LayoutRectangle,
+  ScrollView,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+
+import { Portal, TouchableRipple } from 'react-native-paper';
+import _ from 'lodash';
```

Comment:
Let's use lodash-es and also let's import functions as follow `import { isFunction } from 'lodash-es';`